src/app/containers/ArticleTimestamp/index.test.jsx

```diff
@@ -211,4 +216,25 @@ describe('ArticleTimestamp', () => {
     expect(firstChild(renderedWrapper)).toMatch(regexDate);
     expect(secondChild(renderedWrapper)).toMatch(regexUpdatedDate);
   });
+
+  describe('With different timezones', () => {
+    it('should show the correct local date', () => {
+      const props = {
+        firstPublished: 1565380800000,
+        lastPublished: 1565380800000,
+      };
+      const newsContainer = mount(
+        <WrappedArticleTimestamp {...props} service="news" />,
+      );
+
+      const time = newsContainer.find('time').props().dateTime;
+      expect(time).toEqual('2019-08-09');
```

Comment:
Can this be more specific, like to the hour or something, makes it a bit clearer IMO.
No worries if its huge

Reply:
Nvm, chatted IRL